wp-content/themes/core/assets/js/src/admin/editor/tribe-tinymce.js

```diff
@@ -0,0 +1,64 @@
+( function() {
```

Comment:
Kindly add a DocBlock header explaining the purpose and linking to the related WP Core issue? Future-us will thank you for the references.